src/osemgrep/reporting/Summary_report.ml

```diff
@@ -61,22 +61,26 @@ let pp_summary ppf
   *)
   let opt_msg msg = function
     | [] -> None
-    | xs -> Some (string_of_int (List.length xs) ^ " " ^ msg)
+    | xs ->
+      let file_count = List.length xs in
+      let file_str = if file_count = 1 then "file" else "files" in
```

Comment:
We have a function for that in libs/commons/String_utils.ml call unit_str that you could reuse